### PR TITLE
Add a new question about gene annotations to PBMC3k tutorial

### PR DESCRIPTION
**Pull request by dianichj**
- pbmc3k tutorial: added a question about gene annotations other than Mitochondrial genes
- Added myself as a contributor

### CODE REVIEW

**Review by shiltemann — APPROVED** [review]

File: topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 1]
```diff
@@ -489,6 +490,28 @@ To create this table, we need to:
 >    - *"What to inspect?"*: `Key-indexed annotation of variables/features (var)`
 >
 > 12. Inspect the generated file and check if the mitochondrial annotation has been added
+>
+>    > <question-title></question-title>
+>    >
+>    > 1. How can I filter genes like Ribosomal Protein Genes or Non-Protein Coding genes that might interfere with my downstream analysis?
```
Comment (pavanvidem) [inline topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md:496]:
```suggestion
>    > 1. How can I filter genes like ribosomal protein genes or non-protein coding genes that might interfere with my downstream analysis?
```

File: topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 2]
```diff
@@ -489,6 +490,28 @@ To create this table, we need to:
...
+>    > 2. What steps should I take to identify these genes in my data?
+>    >
+>    > > <solution-title></solution-title>
+>    > >
+>    > > 1. To filter genes such as Ribosomal Protein Genes or Non-Protein Coding genes, follow these steps:
```
Comment (pavanvidem) [inline topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md:501]:
```suggestion
>    > > 1. To filter genes such as ribosomal protein genes or non-protein coding genes, follow these steps:
```

File: topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 3]
```diff
@@ -489,6 +490,28 @@ To create this table, we need to:
...
+>    > >
+>    > >    - Use the Gene annotation file as you did for mitochondrial genes.
```
Comment (pavanvidem) [inline topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md:503]:
```suggestion
>    > >    - Use the gene annotation file as you did for mitochondrial genes.
```

File: topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 4]
```diff
@@ -489,6 +490,28 @@ To create this table, we need to:
...
+>    > >    - Adjust the expression from `$1 ~ /^MT-/` to `$1 ~ /^RP/` for Ribosomal Protein Genes or `$1 ~ /^LINC/` for Non-Protein Coding genes.
```
Comment (pavanvidem) [inline topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md:504]:
```suggestion
>    > >    - Adjust the expression from `$1 ~ /^MT-/` to `$1 ~ /^RP/` for ribosomal protein genes or `$1 ~ /^LINC/` for non-protein coding genes. 
```

File: topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md [hunk 5]
```diff
@@ -489,6 +490,28 @@ To create this table, we need to:
...
+>    > 1. How can I filter genes like Ribosomal Protein Genes or Non-Protein Coding genes that might interfere with my downstream analysis?
+>    > 2. What steps should I take to identify these genes in my data?
+>    >
+>    > > <solution-title></solution-title>
+>    > >
+>    > > 1. To filter genes such as Ribosomal Protein Genes or Non-Protein Coding genes, follow these steps:
+>    > >
+>    > >    - Use the Gene annotation file as you did for mitochondrial genes.
+>    > >    - Adjust the expression from `$1 ~ /^MT-/` to `$1 ~ /^RP/` for Ribosomal Protein Genes or `$1 ~ /^LINC/` for Non-Protein Coding genes. 
+>    > >    -  Create a tabular file for the new annotation column, e.g., rpgene.
```
Comment (pavanvidem) [inline topics/single-cell/tutorials/scrna-scanpy-pbmc3k/tutorial.md:505]:
```suggestion
>    > >    -  Create a tabular file for the new annotation column, e.g., `rpgene` for ribosomal protein genes or `lincrna` for long non-coding RNAs. 
```

**Review by pavanvidem — APPROVED** [review]
Looks good. Congrats on your first contribution and thank you!

**Pull request merged**